coconut.sh

```diff
@@ -0,0 +1,45 @@
+package: coconut
+version: "%(commit_hash)s"
+tag:  master
+build_requires:
+  - golang
+  - protobuf
+source: https://github.com/AliceO2Group/Control
+incremental_recipe: |
+  make WHAT="coconut" all
+  mkdir -p $INSTALLROOT/bin
+  cp bin/* $INSTALLROOT/bin
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+#!/bin/bash -e
+
+GOPATH=$PWD
+PATH=$GOPATH/bin:$PATH
+BUILD=$GOPATH/src/github.com/AliceO2Group/Control
+mkdir -p $BUILD
+rsync -a --delete $SOURCEDIR/ $BUILD/
+cd $BUILD
+make WHAT="coconut" all
+mkdir -p $INSTALLROOT/bin
+cp bin/* $INSTALLROOT/bin
```

Comment:
Make sure you are retaining permissions. Cross-platform way of doing it is not to use `cp` but:

```bash
rsync -a --delete bin/ $INSTALLROOT/bin
```

(mind the final slashes, they matter a lot with `rsync`.)